Another way to mark normativity

xfq opened the pull request:
After discussing it in yesterday's meeting, I tried another way to mark normativity. Please take a look.


<!--
    This comment and the below content is programmatically generated.
    You may add a comma-separated list of anchors you'd like a
    direct link to below (e.g. #idl-serializers, #idl-sequence):

    Don't remove this comment or modify anything below this line.
    If you don't want a preview generated for this pull request,
    just replace the whole of this comment's content by "no preview"
    and remove what's below.
-->
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/i18n-glossary/pull/85.html" title="Last updated on Apr 16, 2026, 2:43 PM UTC (607b760)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/i18n-glossary/85/6b97536...607b760.html" title="Last updated on Apr 16, 2026, 2:43 PM UTC (607b760)">Diff</a>

xfq committed:
Another way to mark normativity

r12a reviewed (changes requested):
LGTM except that the images for 'Glyph' fall outside the box.  I suspect that you need to put the `normative` class on a `div` that surrounds the definition, rather than on the `p` tag.

I assume that you'll also add back the search box you had in your first attempt?

xfq committed:
Fix the 'glyph' entry

xfq commented:
Thanks @r12a. I fixed the 'glyph' entry.

Regarding the search box, I introduced it in #84 primarily because we had two distinct sections, and I didn't want to use alphabetical indexes in both, which would have required handling duplicates. Now that we have a single section, I think it is fine with or without the search box. Since search isn't directly related to normativity, we can discuss it separately.

r12a commented:
As discussed, my suggestion is to keep the search box in addition to the alphabetic links, because it provides a quick way to find uses of a term.  It may also be useful to also highlight uses that are not keywords. (?)

xfq commented:
> As discussed, my suggestion is to keep the search box in addition to the alphabetic links, because it provides a quick way to find uses of a term. It may also be useful to also highlight uses that are not keywords. (?)

I still think this should be discussed separately from this PR, so I created https://github.com/w3c/i18n-glossary/issues/86

xfq commented:
We decided to adopt this approach in last week's teleconference. The next step is to choose which terms are normative and think how to modify the definition. I decided to use this branch (`xfq/normativity-v2`) as a long-term "feature branch" and raise new PRs against this branch.

xfq committed:
Mark 'Mojibake' as informative

per https://www.w3.org/2025/11/09-i18n-minutes.html#7543

xfq committed:
Update the definition of 'Block direction' (#89)

xfq committed:
Update the definition of 'Bidirectional isolate' and 'Bidi isolation' (#88)

xfq committed:
Update the definition of 'Mojibake' (#95)

xfq committed:
Add definition for 'Tofu' (#97)

xfq committed:
Update the definition of 'First-strong detection' (#91)

xfq committed:
Update the definition of 'Incremental time' (#92)
